workbench/icons/48x48/Makefile.am

```diff
@@ -2,6 +2,7 @@ iconsdir = $(datadir)/icons/hicolor/48x48
 icons_appsdir = $(iconsdir)/apps
 
 dist_icons_apps_DATA =							\
+	workbench-basedir.png						\
```

Comment:
Do we really need to ship these? Aren't there any good stock ones? Just wondering

Reply:
Well, I just searched through my installed gnome icons and picked the ones I liked. I did not do any research. Do you now any link to a comfortable gallery of stock icons? I only found C code/a list of defines/names. But no gallery???

Reply:
I found one at http://www.pygtk.org/pygtk2reference/gtk-stock-items.html. But I a, not sure if that is a complete reference.

Reply:
You could also just use the user's icon theme instead of foisting your aesthetics on them :) This icon would be named `user-home`, although I'm not sure that's semantically correct either (it's meant for user's home directory just like this icon).

Reply:
See: https://developer.gnome.org/icon-naming-spec/